src/behat/CentreonContext.php

```diff
@@ -349,6 +350,18 @@ public function launchCentreonWebContainer($name)
             );
         }
     }
+    
+    /**
+     * Set a proxy URL and port
+     */
+    public function setConfiguredProxy()
+    {
+        $proxyConfig = new ParametersCentreonUiPage($this);
+        $proxyConfig->setProperties(array(
+            'proxy_url'=> 'squid',
+            'proxy_port'=> '3128'
+            )); 
```

Comment:
Does not seems PSR-2 compliant. Indentation should be made with the same level than $proxyConfig.

Reply:
sorry